app/assets/stylesheets/views/_answers.scss

```diff
@@ -0,0 +1,9 @@
+.answers {
+  .add-title-margin {
+    @include responsive-top-margin;
+  }
+
+  strong {
```

Comment:
Can we put a comment why this is here since it's inconsistent with the other formats?